src/models/CampaignDb.ts

```diff
@@ -107,7 +117,10 @@ export default class CampaignDb {
                 dataInputPeriods,
                 attributeValues: [
                     { value: "true", attribute: { id: attributeForApp.id } },
-                    { value: dashboard.id, attribute: { id: dashboardAttribute.id } },
+                    {
+                        value: dashboard.id,
+                        attribute: { id: dashboardAttribute.id, code: dashboardAttribute.code },
```

Comment:
[minor] Do we need `code` here? I expected the `id` to be enough to establish the relationship.